Samples/Annotation/annotation.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+// Wrap everything in an anonymous function to avoid polluting the global namespace
+(function () {
+  let worksheets = [];
+  let currentWorksheet;
+
+  $(document).ready(function () {
+    tableau.extensions.initializeAsync().then(function () {
+      worksheets = tableau.extensions.dashboardContent.dashboard.worksheets;
+      if (worksheets.length === 0) {
+        return;
+      }
+      // populating selection menu with worksheets
+      for (let i = 0; i < worksheets.length; i++) {
```

Comment:
you probably know what I'm going to say here ;-)
I won't comment on the rest of the for loops